4-task/config/config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+type DataJson struct {
+	ListenPort string `json:ListenPort`
+	TimeFormat string `json:TimeFormat`
+}
+
+func ReadJsonConfigFile(path string) *DataJson {
```

Comment:
return error if it happens
```suggestion
func ReadJsonConfigFile(path string) (*DataJson, error) {
```